### PR TITLE
Add validation for Account ID and re-enable/add integration tests

### PR DESCRIPTION
**Pull request by ensary**
Re-uses the validator @akhalina wrote for other endpoints and adds tests.

### CODE REVIEW

File: api.yaml [hunk 1]
```diff
@@ -972,7 +972,7 @@ components:
     AccountOwner:
       type: object
       properties:
-        accountId:
+        accountId: # TODO add specific type once we are sure we return it correctly
```
Comment (ensary) [inline api.yaml:975]: I am not 100% sure we stopped returning null here, so not adding the validation here.

**Review by sydneyteh96 — DISMISSED** [review]

**Review by rsoni-atlassian — APPROVED** [review]

**Review by akhalina — APPROVED** [review]

**Comment by akhalina** [comment]
My bad, the build is failing.

File: integration/tests/main_test.go [hunk 2]
```diff
@@ -23,6 +23,7 @@ func TestMain(m *testing.M) {
 	appURL := os.Getenv("AIA_APP_URL")
 	config.BasePath = appURL
 	_, config.Debug = os.LookupEnv("AIA_INTEGRATION_DEBUG")
+	config.Debug = true
```
Comment (ensary) [inline integration/tests/main_test.go:26]: This seems to completely remove any flakes :(

Comment (ensary) [reply integration/tests/main_test.go:26]: I assume forcing I/O after every request prevents one or more race conditions from happening. Let's see if this is true...

File: api.yaml (hunk 1 above)
Comment (akhalina) [reply api.yaml:975]: Yes, I definitely wouldn't touch the account owner stuff for lookups yet.

**Review by akhalina — APPROVED** [review]

**Review by rsoni-atlassian — APPROVED** [review]

**Pull request merged**